config/samples/apps_v1alpha1_pluginconfig.yaml

```diff
@@ -13,4 +13,4 @@ spec:
   pluginName: topology-aware
   pluginVersion: v0.2.3
   state: present
-  # pluginValuesFile: "/path/to/values.yaml"
+  # valuesFile: /tmp/value.yaml
```

Comment:
I am probably missing a large part of the context here. Is this file expected to be found on the node where the operator ends up being scheduled to or on the node where the user is instantiating the operator on ?

Reply:
I'm exploring possible options of loading local file into k8s and so far ConfigMap seems an option to go with but there are some drawbacks too. I will update once I'm satisfied with my current approach that is not pushed here yet.

Reply:
Yes, I also suspect that a volume-mounted inline ConfigMap looks like the only reasonable option for `valuesFile`.

Reply:
@klihub I have now removed the option to pass a file with custom values and instead made it possible to pass it directly via spec.